src/core_scan/Core_scan.ml

```diff
@@ -940,12 +974,10 @@ let mk_target_handler (config : Core_scan_config.t) (valid_rules : Rule.t list)
     (adjusters : Extract.adjusters) match_hook : target_handler =
   (* Note that this function runs in another process *)
   function
-  | `LockfileTarget target ->
-      let (Lockfile_target.{ lockfile; lockfile_kind; _ } as lockfile_target) =
-        lockfile_target_of_input_to_core target
-      in
+  | Lockfile ({ file; kind; _ } as lockfile_location) ->
+      let lockfile_target = lockfile_target_of_location lockfile_location in
       let applicable_supply_chain_rules =
-        select_applicable_supply_chain_rules ~lockfile_kind ~path:lockfile
+        select_applicable_supply_chain_rules ~lockfile_kind:kind ~path:file
```

Comment:
Here we're filtering by path but in the one for code targets we're filtering by source

Reply:
Ah, good catch. I'll update the supply chain stuff too.

Reply:
Fixed in 72603150d9f141f50d0ac51a07fef6c9bd2bc2ce